azure-pipelines/start-emulator.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# This script was copy-pasted from https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/android?view=azure-devops#test-on-the-android-emulator
```

Comment:
not anymore

Reply:
I mentioned `with some changes` to show that it's not a full copy-paste.